api/src/services/monitoring.js

```diff
@@ -0,0 +1,272 @@
+const request = require('request-promise-native');
+
+const db = require('../db');
+const environment = require('../environment');
+const logger = require('../logger');
+
+const DBS_TO_MONITOR = {
+  'medic': environment.db,
+  'sentinel': environment.db + '-sentinel',
+  'usersmeta': environment.db + '-users-meta',
+  'users': '_users'
+};
+
+const getSequenceNumber = seq => {
```

Comment:
The idea is that if you want to track the number of changes that have occurred, say on a chart over time, then it's only the integer component that's useful. This saves the rest of the stack from having to split and parse it.

I could return the actual seq as well if you think it'd be useful?

Reply:
According to CouchDB documentation:
> update_seq (string) – An opaque string that describes the state of the database. Do not rely on this string for counting the number of updates.

Since we don't use the metric for anything important, I guess it's fine.